### PR TITLE
Adding Windows path to the VS Code Debugging setup document.

### PR DESCRIPTION
**Pull request by mirichmo**
Adds additional detail discussed in PR #2204

### CODE REVIEW

**Comment by mirichmo** [comment]
@andschwa and @daxian-dbw Please take a look

**Review by andyleejordan — APPROVED** [review]
Looks good.

**Review by daxian-dbw — DISMISSED** [review]

File: docs/debugging/README.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ addition to being able to build PowerShell, you need:
 - `powershell` executable in your path (self-host if not on Windows)
 
 The .NET CLI tools *must* be on your path for Visual Studio Code.
-`Start-PSBootstrap` installs the tools to `~/.dotnet`,
+`Start-PSBootstrap` installs the tools to `~/.dotnet` (non-Windows) or `"$env:LocalAppData\Microsoft\dotnet"` (Windows),
 but does not add this to your `PATH`.
-You can do this in Bash with `export PATH=$PATH:$HOME/.dotnet`.
+You can do this in Bash with `export PATH=$PATH:$HOME/.dotnet` or in PowerShell with `$env:path = $env:path+";$env:LocalAppData\Microsoft\dotnet"`.
```
Comment (mirichmo) [inline docs/debugging/README.md]: The export command has the same behavior. I just provided the corresponding command for the Windows environment. In both cases, it should be added to the user's profile to make it permanent. PowerShell on Windows also supports direct modification of the registry key to make it system-wide, but that seemed outside the scope of this document.

Comment (daxian-dbw) [inline docs/debugging/README.md]: Sounds reasonable.

**Comment by daxian-dbw** [comment]
CIs are failing due to a dotnet-cli issue. This PR is documentation change, so I will merge it despite the CI failures.

**Pull request merged**